app/models/like.rb

```diff
@@ -1,4 +1,9 @@
 class Like < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  # A method that updates the likes counter for a post
+  def update_likes_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
[OPTIONAL] It is a good practice to use rails callbacks for methods like this. This way you can make the method private thereby improving the security of your backend